app/Providers/RouteServiceProvider.php

```diff
@@ -30,7 +30,7 @@ public function boot()
         $this->registerGlobalRouteParamConstraints();
 
         $this->routes(function () {
-            Route::middleware('api')
```

Comment:
Hum we shouldn't not change the middleware name this should be left as it is - this is a middleware name, not a URL prefix